### PR TITLE
KVM v12.2.0-2020092800 preview release for helm3

### PR DESCRIPTION
**Pull request by tomahawk28**
Re-opening https://github.com/giantswarm/releases/pull/457

Toward giantswarm/giantswarm#11748

### CODE REVIEW

**Comment by tomahawk28** [comment]
Around this comment; https://github.com/giantswarm/releases/pull/457#issuecomment-700712979
@rossf7 @MarcelMue We had been worked with AWS preview release and we use the prefix version as the latest AWS platform release. https://github.com/giantswarm/releases/pull/444/files

so I think we should follow the same method in KVM as well.

**Comment by tomahawk28** [comment]
... and I don't find 12.3.0 KVM release in master branch, did I miss something?

**Comment by tfussell** [comment]
> ... and I don't find 12.3.0 KVM release in master branch, did I miss something?

The idea was that we're creating a prerelease. In semver, v1.2.3-alpha comes before v1.2.3. In this case, we would create a 12.3.0-alpha prerelease based on 12.2.0 (current latest) and then once it has has been tested release it as 12.3.0. What do you think?

**Comment by tomahawk28** [comment]
I like `12.3.0-alpha` naming than the current one, let's see what others think about it.

**Comment by tomahawk28** [comment]
- team-batman decides to go with `12.2.0-2020092800` since this clearly means that we add additional features upon `12.2.0`. We asked team-Ludacris about this naming convention and they also agreed on it. 
- We will add `helm.sh/resource-policy: keep` annotation into release CRs to keep these CRs even after the helm release upgrade.

File: kvm/v12.2.0-2020092800/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+**Note for future 12.x.x releases:**
+
+Please persist this note and the above, until all customers are on AWS v12.2.x and above.
+
+## Change details
+
+### app-operator [v2.3.1](https://github.com/giantswarm/app-operator/blob/master/CHANGELOG.md#231---2020-09-22)
+### chart-operator [v2.3.2](https://github.com/giantswarm/chart-operator/blob/master/CHANGELOG.md#232---2020-09-22)
+- Updated Helm to [v3.3.0](https://github.com/helm/helm/releases/tag/v3.3.4).
```
Comment (rossf7) [inline kvm/v12.2.0-2020092800/README.md:29]: All 3 versions need updating.

**Review by rossf7 — COMMENTED** [review]
Change details section needs to be updated then this looks good.

**Comment by tomahawk28** [comment]
Since we already merged v12.3.0 KVM release with helm3 changes, I'm closing this PR.

**Pull request closed without merging**